finality-aleph/src/import.rs

```diff
@@ -132,24 +120,20 @@ where
             m.report_block(post_hash, Instant::now(), "importing");
         };
 
-        let justifications = block.justifications.take();
+        let justification = block.justification.take();
 
         debug!(target: "afa", "Importing block {:?} {:?} {:?}", number, block.header.hash(), block.post_hash());
-        let import_result = self.inner.import_block(block, cache).await;
+        let import_result = self.inner.import_block(block, cache);
 
         let imported_aux = match import_result {
             Ok(ImportResult::Imported(aux)) => aux,
             Ok(r) => return Ok(r),
             Err(e) => return Err(e),
         };
 
-        if let Some(justification) =
-            justifications.and_then(|just| just.into_justification(ALEPH_ENGINE_ID))
-        {
-            debug!(target: "afa", "Got justification along imported block #{:?}", number);
-
+        if let Some(justification) = justification {
             if self
-                .send_justification(post_hash, number, (ALEPH_ENGINE_ID, justification))
+                .send_justification(post_hash, number, justification)
```

Comment:
Kinda confusing – did we send `ALEPH_ENGINE_ID` together with any justifications and this was completely unnecessary because we actually only ever get this kind of justification anyway, and other ones will not be validated?

Reply:
Or are other types of justifications something that is only introduced in 4.0.0?

Reply:
NVM, found the comment about using GRANDPA justification ID. Question though – I saw `FRNK` in the wallet UI as the justification, what will I see now? Does it matter?

Reply:
I believe it doesn't matter. Now probably it will be `Vec<u8>`.